tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -134,9 +134,12 @@ public static Result IsAwsSqsRequestV0(this MockSpan span) => Result.FromSpan(sp
                 .Matches("aws.agent", "dotnet-aws-sdk")
                 .IsPresent("aws.operation")
                 .IsOptional("aws.region")
+                .IsOptional("region")
```

Comment:
_Is_ this optional? Shouldn't it be required?

Reply:
I think this is set as this due to how the integration tests are created.

Reply:
@zacharycmontoya, this is correct, I can try fix this in another PR, as of now, it seems we were using a non-region config due to easier mocking through a ServiceURL